partiql-eval/src/main/java/org/partiql/eval/Mode.java

```diff
@@ -0,0 +1,38 @@
+package org.partiql.eval;
+
+/**
+ * PartiQL Execution Mode.
+ */
+public class Mode {
+
+    /**
+     * Strict execution mode.
+     */
+    public static final int STRICT = 0;
```

Comment:
Nice! We've been leaving a spot for `UNKNOWN = 0` which _may_ help with protecting against forward compatibility. You don't need to add it here, but you could make `STRICT = 1` and `PERMISSIVE = 2`.